nservicebus/operations/metrics_reporting_metrics_[1.0,1.1).partial.md

```diff
@@ -0,0 +1,29 @@
+## Reporting metrics data
```

Comment:
According to our discussion with @tmasternak I changed the doco, copying 1.0 to 1.1 and warn about getting obsolete.